src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -329,9 +330,21 @@ private function getSchema(): Schema
         return $schema;
     }
 
-    public static function formatDateTime(\DateTimeInterface $dateTime)
+    /**
+     * @throws DBALException
+     *
+     * @deprecated since Symfony 4.4, use convertDateTime instead
+     */
+    public static function formatDateTime(\DateTimeInterface $dateTime, AbstractPlatform $platform)
```

Comment:
adding a required argument is a BC break, which is a bad idea for a method kept as a BC layer...

Reply:
Oh yes, I will add a new commit to fix this.